game-app/game-core/src/main/java/games/strategy/triplea/ai/pro/data/ProTerritoryManager.java

```diff
@@ -160,12 +160,15 @@ private List<ProTerritory> removeTerritoriesThatCantBeConquered(
       final ProTerritory patd = attackMap.get(t);
 
       // Check if I can win without amphib units
-      final List<Unit> defenders =
-          new ArrayList<>(
-              isIgnoringRelationships ? t.getUnitCollection() : patd.getMaxEnemyDefenders(player));
+      final List<Unit> defenders;
+      if (isIgnoringRelationships) {
+        defenders = new ArrayList<>(t.getUnitCollection());
+        defenders.removeAll(patd.getMaxUnits());
```

Comment:
A comment here could be helpful to explain why we are removing the 'patd' units. At a glance it's not clear

Reply:
Done.